assets/styles/styles.css

```diff
@@ -132,6 +132,376 @@ body {
+        border: 1px solid #dedede;
+
+        .feedback__star {
+          width: 2rem;
+          height: 2rem;
+        }
+
+        .item__heading {
+          color: #737373;
+          font-size: 1.4rem;
+          font-weight: 600;
+          margin: 0;
+          line-height: 2.4rem;
+        }
+
+        .feedback__client {
+          display: flex;
+          gap: 15px;
+
+          .client__icn {
+            width: 5rem;
+            height: 5rem;
+          }
+
+          .client__info {
+            font-size: 1.4rem;
+            font-weight: 600;
+            margin: 0;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            gap: 8px;
+
+            .client__link {
+              color: #ffa62b;
+              text-decoration: none;
+            }
+
+            .client__link:hover {
+              text-decoration: underline;
+            }
+
+            .client__position {
+              color: #252b42;
+              margin: 0;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .team {
+    .team__heading {
+      font-size: 4rem;
+      font-weight: 700;
+      margin: 0;
+      padding: 85px 0 17px;
+    }
+
+    .team__subheading {
+      font-size: 1.4rem;
+      font-weight: 500;
+      margin: 0;
+    }
+
+    .team__flex {
+      display: flex;
+      gap: 3.3rem;
+      padding: 81.5px 0;
+
+      .flex__item {
+        display: flex;
+        flex-direction: column;
+
+        .team__info {
+          background-color: #ffffff;
+
+          .team__name {
+            color: #252b42;
+            font-size: 1.6rem;
+            font-weight: 700;
+            margin: 0;
+            padding: 30px 0 5px;
+          }
+
+          .team__position {
+            color: #737373;
+            font-size: 1.4rem;
+            font-weight: 600;
+            margin: 0;
+            padding: 5px 0 9px;
+          }
+
+          .team__socials {
+            list-style: none;
+            display: flex;
+            gap: 1.5rem;
+            margin: 0;
+            padding: 0 0 30px;
+            justify-content: center;
+
+            .socials__link {
+              .link__img {
+                width: 2.4rem;
+                height: 2.4rem;
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .contact {
+    position: relative;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    .container {
+      text-align: left;
+
+      .contact__image {
+        position: absolute;
+        left: 50%;
+        transform: translate(-50%, 0%);
+        z-index: -1;
+        /*width: 100%;*/
+        max-width: 144rem;
+        height: 100%;
+        object-fit: cover;
+      }
+
+      .contact__main {
+        max-width: 60.2rem;
+        .contact__heading {
+          font-size: 1.6rem;
+          font-weight: 700;
+          margin: 0;
+          padding: 90px 0 7px;
```

Comment:
instead of adding padding top here and margin bottom to button, is better to set padding top and bottom to the "contact__main" container